validation/src/main/scala/hmda/validation/rules/lar/validity/V430.scala

```diff
@@ -0,0 +1,18 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V430 extends EditCheck[LoanApplicationRegister] {
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    when((lar.loan.purpose is containedIn(List(1, 2)))) {
```

Comment:
Looks like a typo in the list of loan purpose values. The text of the edit check says "If loan purpose = **2 or 3**, then preapproval must = 3."